### PR TITLE
build(ci): Add Django@4 to tox

### LINKED ISSUE #59: Add support for Django 4.0
Please consider adding support for Django 4.0.
Thanks

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Closes #59

### CODE REVIEW

**Comment by peterbe** [comment]
You might need to drop 3.5. Right? Whatever Django 4 says :)

**Comment by peterbe** [comment]
Yeah, [Django 4 is Python 3.8 and up](https://docs.djangoproject.com/en/4.0/releases/4.0/#python-compatibility)

**Comment by DmytroLitvinov** [comment]
As for me opinion, I would do it as other packages in Django eco system do: they provide new release with support only for Django 2.2, 3.1, 3.2, 4.0 and Python 3.6^.

How do you see it, @peterbe ?

**Comment by peterbe** [comment]
> As for me opinion, I would do it as other packages in Django eco system do: they provide new release with support only for Django 2.2, 3.1, 3.2, 4.0 and Python 3.6^.

I'm not entirely sure I understand you. But I'll phrase it myself and you can check if we are saying the same thing. 

If someone needs to stick to Django 2, they can simply pin an old version of `django-cache-memoize`. 
If we add support for Django 4.0 we'll just make a semver release and a big mention in the changelog that this project now only supports Python 3.8 and up.

**Comment by DmytroLitvinov** [comment]
And why we can't just add Django@4 in the tox.ini without dropping Django 2.2,3.1,3.2? We only need use Python 3.8^ for Django@4.0. 

My proposal is to add Django@4.0 right now without any release or something. There is no code changes or something and even we follow semver, we can add new patch release where we point out that we added Django@4.0 to tox.ini for testing that package. (But again - without code changes in the library I would not do release). This is more for us right now to see that current version of package can be installed for Django@4.0

File: tox.ini [hunk 1]
```diff
@@ -6,9 +6,9 @@ envlist =
     py35-django{22},
     py36-django{22,31,32},
     py37-django{22,31,32},
-    py38-django{22,31,32},
-    py39-django{22,31,32},
-    pypy3-django{22,31,32},
+    py38-django{22,31,32,40},
+    py39-django{22,31,32,40},
+    pypy3-django{22,31,32,40},
```
Comment (peterbe) [inline tox.ini:11]: Perhaps pypy doesn't have a Django4. So perhaps drop this line?

**Review by peterbe — APPROVED** [review]
Thanks!

**Pull request merged**